### ISSUE TITLE
MenuItem- Make event handler lambda optional

### ISSUE THREAD

**Issue by thomasnield** [edvin/tornadofx#43]
I've been working on RxJavaFX to [implement reactive streams for `ActionEvent`](https://github.com/ReactiveX/RxJavaFX/blob/0.x/src/main/java/rx/javafx/sources/ActionEventSource.java), including Menus. I think it might be helpful to make the event handler for `menuitem` optional, especially for us Rx folks to use `Observable` or `EventStream`.

So instead of having to write this with a placeholder `{ }`

```kotlin
menuitem("Press Me") { }
      .actionEvents().map { buildNewList() }.subscribe { myTable.items = it }
```

I can do this instead. 

```kotlin
menuitem("Press Me")
      .actionEvents().map { buildNewList() }.subscribe { myTable.items = it }
```

**Comment by edvin** [comment]
Absolutely, just make it a noop like we do with some of the builder constructs :) Sorry for not getting back to you on the other matters Thomas, busy day!

**Comment by thomasnield** [comment]
No problem!

**Comment by thomasnield** [comment]
Resolved in #49
